README.md

```diff
@@ -14,7 +15,36 @@ CodeFlare Stack Compatibility Matrix
 | KubeRay                      | v0.5.0  |
 <!-- Compatibility Matrix end -->
 
-## Release process
+## Development
+
+### Testing
+
+The e2e tests can be executed locally by running the following commands:
+
+1. Setup the test cluster:
+
+    ```bash
+    # Create a KinD cluster
+    $ kind create cluster --image kindest/node:v1.25.8
+    # Install the CRDs
+    $ make install
+    ```
+
+2. Start the operator locally:
```

Comment:
And what about starting MCAD (https://github.com/project-codeflare/codeflare-operator/blob/ce9349577536ed59d2f66c7b57550f64a79b74a7/.github/workflows/e2e_tests.yaml#L116-L164)?
Isn't it also required for test execution?

Reply:
And Kuberay?

Reply:
Ah right, these two steps are missing 👍🏼.

Reply:
Done with ff32292c1eba52174a5eb248f4fa56144c79aba8.